src/features/hotspots/root/HotspotsView.tsx

```diff
@@ -129,6 +129,14 @@ const HotspotsView = ({
     return [1, listHeight.current]
   }, [detailHeaderHeight, listHeight, showDetails])
 
+  const handleMapLoaded = useCallback(() => {
```

Comment:
The map is a beast and causes jankiness - wait for it to load to reveal the list.